api_tests/registrations/views/test_registration_detail.py

```diff
@@ -66,3 +69,44 @@ def test_do_not_return_node_detail(self):
         res = self.app.get(url, auth=self.user.auth, expect_errors=True)
         assert_equal(res.status_code, 400)
         assert_equal(res.json['errors'][0]['detail'], "This is not a registration.")
+
+    def test_retractions_display_limited_fields(self):
+        registration = RegistrationFactory(creator=self.user, project=self.public_project, public=True)
+        url = '/{}registrations/{}/'.format(API_BASE, registration._id)
+        retraction = RetractedRegistrationFactory(registration=registration, user=registration.creator)
+        retraction.justification = 'We made a major error.'
+        retraction.save()
+        res = self.app.get(url, auth=self.user.auth, expect_errors=True)
+        assert_equal(res.status_code, 200)
+
+        assert_items_equal(res.json['data']['attributes'], {
```

Comment:
`assert_items_equal` only compares the counts of two sequences (lists, tuples, etc.): https://docs.python.org/2/library/unittest.html?highlight=assertitemsequal#unittest.TestCase.assertItemsEqual . Use `assert_equal` for deep comparison of dictionaries.